tests/Paket.Tests/PackageProcessSpecs.fs

```diff
@@ -11,7 +11,7 @@ let ``Loading assembly metadata works``() =
     let workingDir = Path.GetFullPath(".")
     
     let fileName =
-        Path.Combine(workingDir, "..", "..", "Paket.Tests.fsproj")
+        Path.Combine(workingDir, "Paket.Tests.fsproj")
```

Comment:
I'm not quite sure, it just fails under NUnit3 without that as it looks in the wrong folder.